build.gradle

```
@@ -180,9 +184,16 @@ void createTestTask(String name, String environments, boolean devMode, int javaV
 			'src/test/skript/tests',
 			'src/test/resources/runner_data',
 			environments,
-			devMode,
-			genDocs
+			modifiers.contains(Modifiers.DevMode),
+			modifiers.contains(Modifiers.GenDocs),
+			modifiers.contains(Modifiers.Debug),
+			project.findProperty('verbosity') ?: "null"
 		]
+
+		if (modifiers.contains(Modifiers.Profile)) {
+			if (project.property('profiler') == null) logger.error('Add parameter -Pprofiler=<path to profiler>')
+			else args += '-agentpath:' + project.property('profiler') + '=port=8849,nowait'
```

It may be .gradle but I think it should follow our Code Conventions still (tho I don't know the syntax for groovy exactly)
```suggestion
			if (project.property('profiler') == null) {
				logger.error('Add parameter -Pprofiler=<path to profiler>')
			 } else {
			 	args += '-agentpath:' + project.property('profiler') + '=port=8849,nowait'
			 }
```